### DISCUSSION TITLE
.macOS(.v15)+

### DISCUSSION THREAD

**Discussion by shrutig2720** [swift-cloud/Vercel#32] [Q&A]
How can I increase the platform version that vercel is running on... it is currently set to .v12 and the Xcode project I have needs at least .v15? Any advice would be appreciated.

**Comment by mcritz** [comment]
Open the `Package.swift` file and change the `platforms`. Ex:

```
[.macos(.v15)]
```

**Reply by shrutig2720** [reply]
Thanks for the response! I was trying to make it work for an ios app and it seems that package.swift is abstracted for the ios apps. I did find a work around by changing build settings, but couldn’t vercel link bc it doesn’t have package.swift!
